<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1724</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>1251</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>1251</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget_2" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>283</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>通道数</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>采样时钟</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>触发模式</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>触发源</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分频因子</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>触发电平</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>触发延时</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>触发长度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>输入增益</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QComboBox" name="lChCnt">
      <item>
       <property name="text">
        <string>1通道</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2通道</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="m_bSelClk">
      <item>
       <property name="text">
        <string>内时钟</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>外时钟</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="TriggerMode">
      <item>
       <property name="text">
        <string>连续采集</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>后触发</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>延时触发</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="TriggerSource">
      <item>
       <property name="text">
        <string>外正沿</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>外负沿</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>软件触发</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>内正沿</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>内负沿</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="m_nClkdeci">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TriggerLevel">
      <property name="text">
       <string>2048</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TriggerDelay">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TriggerLength">
      <property name="text">
       <string>1024</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lADGain">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="m_bInt">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>计数器</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_bEn2G">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2G模式</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>累加功能</string>
    </property>
    <widget class="QCheckBox" name="bEnADD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>使能</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_6">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>72</y>
       <width>151</width>
       <height>54</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>次数</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>门限</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLineEdit" name="lADDcnt">
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lADDthd">
       <property name="text">
        <string>255</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>实时采集</string>
    </property>
    <widget class="QPushButton" name="startAD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>启动</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopAD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>结束</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>单次采集</string>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>30</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>长度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_lsicnt">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="singleAD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>单次采集</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>30</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>MS</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>保存路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="savePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectPath">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>580</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
